examples/c/mass_spring_example.c

```diff
@@ -128,14 +137,17 @@ int main() {
                     #endif
                     ((ocp_qp_partial_condensing_args *)((ocp_qp_sparse_solver_args *)args)->pcond_args)->N2 = N2;
                     break;
+                #endif
                 case FULL_CONDENSING_HPIPM:
                     printf("\nFull condensing + HPIPM:\n\n");
                     // default options
                     break;
+                #if ACADOS_WITH_QORE
```

Comment:
This #ifdef is superfluous I think.

Reply:
This one indeed, but only because we don't change any options of qore, was more like for the shake of completeness